src/jit/gentree.h

```diff
@@ -5535,22 +5535,21 @@ struct GenTreeCopyOrReload : public GenTreeUnOp
 
     unsigned GetRegCount()
     {
-        if (gtRegNum == REG_NA)
-        {
-            return 0;
-        }
 #if FEATURE_MULTIREG_RET
-        for (unsigned i = 0; i < MAX_RET_REG_COUNT - 1; ++i)
+        // We need to return the highest index for which we have a valid register.
```

Comment:
So there can be cases where `gtOtherRegs[0] == REG_NA` but `gtOtherRegs[1] != REG_NA`? I don't understand how that would happen, but nonetheless, it seems there should be a comment explaining this can happen. I thought a lot of these various multireg functions assumed there were no "holes" in the register sets.

Reply:
@BruceForstall - The case with "holes" can only be true for the COPY or RELOAD - these are the cases where only the registers that need the be handled are something other than `REG_NA`. I'll add a comment.